src/Components/NewContact/index.js

```diff
@@ -1,11 +1,104 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import { getDownloadURL, listAll, ref, uploadBytes } from "firebase/storage";
+
+import { fireStorage } from "../../Config/firebaseInit";
+
+import setContacts from "../../Services/setContacts";
+
+import styles from "./index.module.scss";
+
+const Form = () => {
+    const [state, setState] = useState({
+        firstName: "",
+        lastName: "",
+        phone: "",
+        email: "",
+        avatar: "",
+    });
+
+    const handleInputChange = (e) => {
+        setState({
+            ...state,
+            [e.target.name]: e.target.value,
+        });
+    };
+
+    useEffect(() => {
+        setContacts(state)
+        alert("Contact is ready")
+    }, [state.avatar]);
+
+    const handleSetContact = (e) => {
+        e.preventDefault();
+        const imageRef = ref(fireStorage, `contactAvatars/${e.target[4].files[0].name}`);
+            uploadBytes( imageRef, e.target[4].files[0] ).then(() => {
+            const imageListRef = ref(fireStorage, `contactAvatars/`);
+            listAll(imageListRef).then((response) => {
+                response.items.forEach((item) => {
+                    if (item.name === e.target[4].files[0].name) {
+                        getDownloadURL(item).then((uploadPicUrl) => {
+                          setState({
+                                ...state,
+                                avatar: uploadPicUrl,
+                            });
+                        });
+                    };
+                });
+            });
+        });
+    };
+
 
-const NewContact = () => {
     return (
-        <div className="newContact">
-            New contact
-        </div>
+        <form onSubmit={handleSetContact} className={styles.newContact}>
+            <label className={styles.label}>
+                Name: 
+                <input
+                    type="text"
+                    name="firstName"
+                    value={state.firstName}
+                    onChange={handleInputChange}
+                />
+            </label>
```

Comment:
This format is duplicated. I mean there is many inputs and their difference is just in some props. You can create a constant array and fill it with objects like this one
[
  {
    name: 'firstName',
    type: 'text',
    label: 'Name: ',
  },
]
and then write a map() and render all these objects.
I mean all these inputs beside avatar